koku/api/tags/serializers.py

```diff
@@ -175,3 +175,8 @@ def validate_filter(self, value):
         """
         validate_field(self, 'filter', AWSFilterSerializer, value)
         return value
+
+
+class OCPAWSTagsQueryParamSerializer(AWSTagsQueryParamSerializer,
```

Comment:
Should only need to subclass the AWS Serializer.

```suggestion
class OCPAWSTagsQueryParamSerializer(AWSTagsQueryParamSerializer)
```

Reply:
That is incorrect. We need to subclass from both to get the union of serializer fields. Otherwise, we'd have no way to validate the fields across both AWS and OCP datasets.

Reply:
After discussion, I agree that it's a good thing to have the ability to filter on AWS account or OpenShift project for this API, but the multiple inheritance situation is causing 

```
GET /local/v1/tags/openshift/infrastructures/aws/?filter[project]=1
HTTP 400 Bad Request
Allow: GET, OPTIONS
Content-Type: application/json
Vary: Accept

{
    "filter": {
        "project": [
            "Unsupported parameter or invalid value"
        ]
    }
}
```

Reply:
I've pushed a fix for this. It passes validation now:

```
COMMAND: curl -v -g -H 'HTTP_X_RH_IDENTITY: ImlkZW50aXR5Ijp7ImFjY291bnRfbnVtYmVyIjoiMTAwMDEiLCJ0eXBlIjoiVXNlciIsInVzZXIiOnsidXNlcm5hbWUiOiJ0ZXN0X2N1c3RvbWVyIiwiZW1haWwiOiJibGVudHoraGNjbS10ZXN0QHJlZGhhdC5jb20ifX0K' http://localhost:8000/api/v1/tags/openshift/infrastructures/aws/?filter[project]=abc
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0* Could not resolve host: ImlkZW50aXR5Ijp7ImFjY291bnRfbnVtYmVyIjoiMTAwMDEiLCJ0eXBlIjoiVXNlciIsInVzZXIiOnsidXNlcm5hbWUiOiJ0ZXN0X2N1c3RvbWVyIiwiZW1haWwiOiJibGVudHoraGNjbS10ZXN0QHJlZGhhdC5jb20ifX0K'
* Closing connection 0
curl: (6) Could not resolve host: ImlkZW50aXR5Ijp7ImFjY291bnRfbnVtYmVyIjoiMTAwMDEiLCJ0eXBlIjoiVXNlciIsInVzZXIiOnsidXNlcm5hbWUiOiJ0ZXN0X2N1c3RvbWVyIiwiZW1haWwiOiJibGVudHoraGNjbS10ZXN0QHJlZGhhdC5jb20ifX0K'
*   Trying ::1...
* TCP_NODELAY set
* connect to ::1 port 8000 failed: Connection refused
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 8000 (#1)
> GET /api/v1/tags/openshift/infrastructures/aws/?filter[project]=abc HTTP/1.1
> Host: localhost:8000
> User-Agent: curl/7.64.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Date: Thu, 30 May 2019 17:55:19 GMT
< Server: WSGIServer/0.2 CPython/3.7.3
< Content-Type: application/json
< Vary: Accept, Origin
< Allow: GET, OPTIONS
< X-Frame-Options: SAMEORIGIN
< Content-Length: 378
< 
{ [378 bytes data]
100   378  100   378    0     0    818      0 --:--:-- --:--:-- --:--:--   818
* Connection #1 to host localhost left intact


{
  "meta": {
    "count": 0,
    "filter": {
      "project": [
        "abc"
      ],
      "time_scope_value": "-10",
      "time_scope_units": "day",
      "resolution": "daily"
    },
    "key_only": false
  },
  "links": {
    "first": "/api/v1/tags/openshift/infrastructures/aws/?filter%5Bproject%5D=abc&limit=100&offset=0",
    "next": null,
    "previous": null,
    "last": "/api/v1/tags/openshift/infrastructures/aws/?filter%5Bproject%5D=abc&limit=100&offset=0"
  },
  "data": []
}

```